sbi/utils/conditional_density_utils.py

```diff
@@ -294,7 +294,7 @@ def __init__(
         self.device = self.potential_fn.device
         self.allow_iid_x = allow_iid_x
 
-    def __call__(self, theta: Tensor, track_gradients: bool = True) -> Tensor:
+    def __call__(self, theta: Tensor, track_gradients: bool = True, x_o=None) -> Tensor:
```

Comment:
For consistency, I think that `x_o` should be the second argument, no? Also please add type-hint.